src/app/window-handler.ts

```diff
@@ -259,27 +247,38 @@ export class WindowHandler {
             // Injects custom title bar css into the webContents
             // only for Window and if it is enabled
             await injectStyles(this.mainWindow, this.isCustomTitleBarAndWindowOS);
-            if (this.isCustomTitleBarAndWindowOS) {
-                this.mainWindow.webContents.send('initiate-custom-title-bar');
-            }
 
             this.mainWindow.webContents.send('page-load', {
                 isWindowsOS,
                 locale: i18n.getLocale(),
                 resources: i18n.loadedResources,
                 origin: this.globalConfig.url,
+                enableCustomTitleBar: this.isCustomTitleBarAndWindowOS,
```

Comment:
do we need such a big name `isCustomTitleBarAndWindowOS` given that custom title bar is only applicable for windows. Can we change it to just `this.isCustomTitleBar`?